packages/platform-fastify/constants.ts

```diff
@@ -1,3 +1,4 @@
 export const FASTIFY_ROUTE_CONFIG_METADATA = '__fastify_route_config__';
 export const FASTIFY_ROUTE_CONSTRAINTS_METADATA =
   '__fastify_route_constraints__';
+export const FASTIFY_SCHEMA_METADATA = '__fastify_schema__';
```

Comment:
```suggestion
export const FASTIFY_SCHEMA_METADATA = '__fastify_route_schema__';
```

let's be more specific.

Reply:
Happy to oblige